docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+.. _backends_guide:
+
+==============
+Backends guide
+==============
+
+Backends are the interfaces used to run :ref:`Circuits <circuits_guide>` in Orquestra Core. In this guide we'll cover differences between types of backends (``QuantumBackend`` vs ``QuantumSimulator``, ``TrackingBackend``, ``SymbolicSimulator``), how to use backends, currently integrated backends, and creating your own backend (among other topics).
```

Comment:
I would drop `TrackingBackend` and `SymbolicSimulator` from here.

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64